source/run-command/tests/unified/runCommand.yml

```diff
@@ -67,6 +67,36 @@ tests:
                 $readPreference: { $$exists: false }
               commandName: ping
 
+  - description: always gossips the $clusterTime on the sent command
```

Comment:
There is a test case in this file with description "does not inherit writeConcern specified at the db level". This TC will produce the following error indicated a duplicate _id:

```
unified_spec_runner.go:134: error running operation "runCommand" at index 0: error verification failed: expected no error, but got write exception: write errors: [E11000 duplicate key error collection: dbWithWC.collection index: _id_ dup key: { _id: 1 }]
```

Should we just not specify and _id here and insert a auto-incr document such as `{"foo":"bar"}`?

Reply:
Sure, good improvement, not sure why I wasn't seeing the same error, must be some clean db reset 🤔